Cabal/Distribution/Simple/Setup.hs

```diff
@@ -110,6 +107,10 @@ import Distribution.Compat.Semigroup (Last' (..))
 
 import Data.Function (on)
 
+-- To be removed
```

Comment:
Is there a tracking bug for this, explaining when it'll happen/what it's blocked on?

Reply:
#5580

Reply:
OK, so they're going away in a later revision of this PR? That's fine too. I just don't want mystery 'to be removed' comments living in the code with no indication of when or how that's meant to happen.

Reply:
... in the follow up PR.